docs/schemas/resource/IfcGeometricConstraintResource/Entities/IfcAlignmentVerticalSegment.md

```diff
@@ -7,9 +7,9 @@ The vertical alignment is defined by segments that connects end-to-start. The ve
 
 The following vertical segment types are defined:
 
-* line segment - _IfcAlignmentVerticalSegmentTypeEnum_ .LINE.
-* circular arc segment - _IfcAlignmentVerticalSegmentTypeEnum_ .ARC.
-* parabolic arc segment - _IfcAlignmentVerticalSegmentTypeEnum_ .PARABOLICARC. which can describe symmetric parabolas, unsymmetric parabolas can be created by defining two symmetric parabolas segments
+* line segment - _IfcAlignmentVerticalSegmentTypeEnum_ .CONSTANTGRADIENT.
+* circular arc segment - _IfcAlignmentVerticalSegmentTypeEnum_ .CIRCULARARC.
+* parabolic arc segment - _IfcAlignmentVerticalSegmentTypeEnum_ .PARABOLICARC. which can describe symmetric parabolas. Unsymmetric parabolas can be created by defining two symmetric parabolas segments
```

Comment:
Asymmetric? Or is this specific idiom?

Reply:
To my knowledge the terms are interchangeable.